docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
+1 on `shutdown`. @morphis and @tonyespy - please feel free to move forward on renaming.